[ZEPPELIN-6265] Support classic Helium viz packages in new UI

tbonelee opened the pull request:
### What is this PR for?
This PR enables the use of external Helium visualization packages, which are previously available only in the classic UI, in the new UI.

The process of loading and importing Helium packages, as well as rendering them, closely follows the logic from the classic UI.

Since the new-style visualization classes in the new UI are not compatible with the existing ones, the implementation branches accordingly to handle both.

Classic Helium visualization packages depend on legacy technologies such as AngularJS, jQuery, and several visualization libraries, which had to be included as a result.
For Bootstrap styles, conflicts arose with existing styles, so I copied and modified the original HTML templates to prevent clashes.
If a Helium package attempts to use the original HTML template, it will be replaced with the modified version.

Additionally, I identified that the Helium visualization support classes from the discontinued development of the new UI are no longer in use, so I removed them.

### What type of PR is it?

Improvement

### Todos
* [v] - Added support for loading Helium packages
* [v] - Fixed style mismatches in Helium packages
* [v] - Other functional issues resolved

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-6265

### How should this be tested?
* Enables a classic Helium visualization package in the classic UI, and verify that it works in the new UI.
  - Note: The `/helium` package management page has not yet been added.

### Questions:
* Does the license files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

seung-00 commented on zeppelin-web-angular/src/app/services/angular-drag-drop.service.ts:
```diff
@@ -0,0 +1,328 @@
+    module.directive('jqyouiDraggable', [
+      'customDragDropService',
+      // tslint:disable-next-line:no-any
+      function(dragDropService: any) {
+        return {
+          restrict: 'A',
+          // tslint:disable-next-line:no-any
+          link: function(scope: any, element: any, attrs: any) {
```
How about removing the explicit `any`? Type inference would work correctly here.

tbonelee commented on zeppelin-web-angular/src/app/services/table-data-adapter.service.ts:
```diff
@@ -0,0 +1,115 @@
+        const updatedClassicData = this.convertToClassicFormat(modernTableData);
+        this.columns = updatedClassicData.columns;
+        this.rows = updatedClassicData.rows;
+        this.comment = updatedClassicData.comment;
```
You're right, the `this` binding was incorrect here. So I changed it to use arrow functions and access through variable names instead.

tbonelee commented:
@seung-00 I've checked for places where type inference can be used as suggested in the review, and fixed the incorrect `this` bindings. I've also added proper type definitions to replace `any` usage in several places.

Reamer commented on zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java:
```diff
@@ -359,11 +359,11 @@ private void prepareSource(HeliumPackage pkg, String[] moduleNameVersion,
-    loadJsRegister.append("icon: " + gson.toJson(pkg.getIcon()) + ",\n");
-    loadJsRegister.append("type: \"" + pkg.getType() + "\",\n");
-    loadJsRegister.append("class: " + className + "\n");
+    loadJsRegister.append("  id: \"" + moduleNameVersion[0] + "\",\n");
```
My suggestion is to use the following style so that no string concatenation is performed.
```suggestion
    loadJsRegister.append(idention).append("id: \"").append(moduleNameVersion[0]).apend("\",\n");
```

Of course, the idention variable must be defined before, but it also makes it easier to change it globally. It also makes the code more readable, explaining why two spaces are inserted.  I don't think these are necessary from a purely programmatic point of view, do they?

tbonelee replied:
Thanks for pointing that out. I've removed the indentation (as it didn't really improve readability) and replaced the string concatenation with `.append()`, following your suggestion.

Reamer reviewed (commented):
A minor comment on the backend code. I am unable to review the frontend part.

tbonelee commented:
Rebased onto the master branch and add some Prttier exclusion rules for `*.min.css` and `*.min.js` files

ParkGyeongTae commented:
@tbonelee Please rebase onto the latest master to resolve the conflicts.

tbonelee commented:
Rebased it onto the master branch

tbonelee commented:
Thanks for the reviews, I've merged this into master and branch-0.12

dididy commented:
It looks like zeppelin-angular-web/src/app/helium-manager/public-api.ts is an empty file and is missing the license header comment, or it might not have been removed.
